client/src/redux/middleware/index.js

```diff
@@ -0,0 +1,12 @@
+
+export const junk = (/*store*/) => next => action => {
```

Comment:
what is the use case of this code?

Reply:
For learning / researching , forgot to delete. 
Removed. https://github.com/Levayv/ipma_v2/commit/e59e42e0f616b2de8d0c48f3f1c311eea0ab8a53#diff-b8b9df5809f3d4a76c86647f72a90d51L1